keyboards/handwired/sauceball/keymaps/default/keymap.c

```diff
@@ -0,0 +1,57 @@
+// Copyright 2023 xia0 (@xia0)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "print.h"
+
+/* CONFIG */
+// modify movement rates as a percentage
+// use this to reduce mouse sensitivity especially for drag-scrolling
+const int SENSITIVITY_X = 40;
+const int SENSITIVITY_Y = 40;
+const int SENSITIVITY_H = 3;
+const int SENSITIVITY_V = 3;
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_mouse(KC_BTN1, KC_BTN3, KC_BTN4, KC_BTN2)
+};
+
+int move_counters[4]; // array to keep count of "incomplete" moves for low sensitivity settings
```

Comment:
The default keymap should be kept simple and minimal. 

This should be moved into another keymap, or into the keyboard's c file (eg, sauceball.c)

Reply:
Will do. I'm away for a few weeks so I'll do it when I get back and can test on hardware. I'll revert this PR to draft.

Reply:
I'll add an "on hold" label for you. Let us know when you have it working.

Reply:
> I'll add an "on hold" label for you. Let us know when you have it working.

The default keymap works fine on hardware.

The xia0 keymap works too but I've disabled a custom key for drag-scrolling for the time being because I cannot figure out why tap_code(KC_BTN#) does not work but works for other keycodes within process_record_user.